charts/linkerd-control-plane/values.yaml

```diff
@@ -544,4 +546,4 @@ podMonitor:
     enabled: true
   proxy:
     # -- Enables the creation of PodMonitor for the data-plane
-    enabled: true
+    enabled: true
```

Comment:
Can you please revert the removal of the trailing newline?

Reply:
Done.